### DISCUSSION TITLE
Merge non-capitalized / non-exact match UTM tags with organic traffic too

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#2153] [Anything Else]
Most marketing teams use all lowercase UTM tags so the fact that [we only merge capitalized UTM tags with the organic sources](https://plausible.io/docs/manual-link-tagging#merging-organic-traffic-and-paid-campaigns) creates a bit of an issues for them when switching to Plausible. They need to either continue using all lowercase but then seeing say "Google" and "google" separate entries in the Top Sources report or switching all campaigns to use exact names which for many is difficult as it's a part of a bigger process. Would be nice if we smooth this onboarding process and start merging non-exact match and non-capitalized UTM sources too with the organic traffic.

**Comment by hosti** [comment]
ohh, I already know why I have facebook twice on "Top Sources" 
Thanks :)
